extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/build/OtelBuildConfig.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.opentelemetry.runtime.config.build;
+
+import static io.quarkus.opentelemetry.runtime.config.build.PropagatorType.Constants.BAGGAGE;
+import static io.quarkus.opentelemetry.runtime.config.build.PropagatorType.Constants.TRACE_CONTEXT;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * Build Time configuration where all the attributes related with
+ * classloading must live because of the native image needs
+ */
+@ConfigRoot(name = "otel", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public final class OtelBuildConfig {
+    public static final String INSTRUMENTATION_NAME = "io.quarkus.opentelemetry";
+
+    /**
+     * If false, disable the OpenTelemetry usage at build time. All other Otel properties will
+     * be ignored at runtime.
+     * <p>
+     * Will pick up value from legacy property quarkus.opentelemetry.enabled
+     * <p>
+     * Defaults to true.
+     */
+    @Deprecated // TODO only use runtime (soon)
+    @ConfigItem(defaultValue = "${quarkus.opentelemetry.enabled:true}")
+    public boolean enabled;
+
+    /**
+     * Trace exporter configurations
+     */
+    public TracesBuildConfig traces;
+
+    /**
+     * No Metrics exporter for now
+     */
+    @ConfigItem(name = "metrics.exporter", defaultValue = "none")
```

Comment:
So you will have a singleton list with `none` in it, right?

Reply:
Yes. This will effectively disable OTel metrics.